### DISCUSSION TITLE
How to see the hotspot in ar mode android

### DISCUSSION THREAD

**Discussion by GabrielAllbreaker** [google/model-viewer#3743] [General]
Hello there, im trying to see the hotspot in the ar mode but in dont see that.
 in the examples y can see that but when im triying to employ that in my web this don't Works
the code i use is this and i use a css but when i charge the ar mode i cant see the hotspots

can anyone help me?

https://drive.google.com/drive/folders/1p0SQrcRIJhWdMPxiUoeiwsOpVlCrNBC-?usp=sharing

**Comment by elalish** [comment]
Hotspots only work in WebXR mode (the other modes leave the browser). You can verify you're in WebXR mode by setting `ar-modes="webxr"`. If AR doesn't work at all then, check your console logs for info. Usually it's that you need to serve on HTTPS.

**Reply by GabrielAllbreaker** [reply]
https was the solution, thanks, I was trying to see the hotspot on localhost, but uploading my example to the server this works!
